website/pages/blog/announcing-the-vercel-source-plugin.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Announcing The Vercel Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Vercel source plugin.
+author: kemal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Gandi source plugin for CloudQuery.
+
+Vercel is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration.
```

Comment:
I think we can skip this and maybe say `If you're not familiar with the Vercel platform, we recommend you visit their [homepage](https://vercel.com/)` instead of coping their tagline.